res/values-sq/strings.xml

```diff
@@ -247,122 +247,122 @@
     <string name="Found_a_single_player_stats_backup_with_higher_XP__Do_you_want_to_restore_the_backup_">"Gjetur një lojtar të vetëm stats backup me XP të lartë A doni të rivendosur backup?"</string>
 
     <string name="Special_Effects">"Efekte speciale"</string>
-    <string name="Blob_Mushiness">"Blob Mushiness"</string>
+    <string name="Blob_Mushiness">"Njollë pucrra"</string>
 
     <string name="Specify_private_game_name">"Specifikoni emrin Game privat"</string>
-    <string name="In_Game_Chat">"Në lojë chat"</string>
-    <string name="My_Status_">"Statusi im:"</string>
-    <string name="Game_Size_">"Lojë Size:"</string>
+    <string name="In_Game_Chat">"Në lojë Chat"</string>
+    <string name="My_Status_">"Statusi imë:"</string>
+    <string name="Game_Size_">"Madhësia i lojës:"</string>
 
-    <string name="Server_Restart_">"Server Restart:"</string>
+    <string name="Server_Restart_">"Rifillimi Serverit:"</string>
     <string name="GOAL_">"GOL!"</string>
     <string name="SOCCER">"FUTBOLL"</string>
     <string name="DRAW">"BARAZIM"</string>
 
     <string name="Clan_Name">"Emri Klan"</string>
-    <string name="Clan">"Fis"</string>
+    <string name="Clan">"Klan"</string>
 
 
-    <string name="JOIN_CLAN">"JOIN FIS"</string>
-    <string name="JOIN_RANDOM_CLAN">"bashkohet me fis rastit"</string>
-    <string name="LEAVE_CLAN">"lënë klanin"</string>
-    <string name="INVITE_MEMBER">"Fto anëtar"</string>
-    <string name="REMOVE_MEMBER">"Hiq anëtarin"</string>
+    <string name="JOIN_CLAN">"HINI NË KLAN"</string>
+    <string name="JOIN_RANDOM_CLAN">"BASHKOHU NË KLAN E RASTIT"</string>
+    <string name="LEAVE_CLAN">"LËNË KLANIN"</string>
+    <string name="INVITE_MEMBER">"FTO ANËTARIN"</string>
+    <string name="REMOVE_MEMBER">"HIQNI ANËTARIN"</string>
 
 
-    <string name="CREATE_CLAN">"KRIJO FIS"</string>
+    <string name="CREATE_CLAN">"KRIJO KLAN"</string>
     <string name="MEMBER">"ANËTAR"</string>
     <string name="ADMIN">"ADMIN"</string>
-    <string name="LEADER">"LEADER"</string>
+    <string name="LEADER">"LIDER"</string>
     <string name="ELDER">"ELDER"</string>
 
 
-    <string name="Not_in_a_clan_">"Jo në një fis."</string>
+    <string name="Not_in_a_clan_">"JO NË NJË KLAN."</string>
```

Comment:
Please return the small letters as they were before. These aren't supposed to be 100% capitals.